website/docs/guides/Programmatic-API-Key-upgrade-guide-1.10.0.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: "mongodbatlas"
+page_title: "Upgrade Guide for Terraform MongoDB Atlas Provider Programmatic API Key Resource in v1.10.0"
+sidebar_current: "docs-mongodbatlas-guides-Programmatic-API-Key-upgrade-guide"
+description: |-
+MongoDB Atlas Provider : Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider: Programmatic API Key Upgrade Guide in v1.10.0
+In Terraform MongoDB Atlas Provider v1.10.0, some improvements were introduced which mainly focus on the MongoDB Atlas Programmatic API Keys (PAK) handling. The improvements introduce a much more sustainable way to manage PAK project assignment lifecycle over the long term versus being embedded in the project resource.  This guide will help you to transition smoothly from the previous version which this resource was first released (v1.8.0) to the new version (v1.10.0).
```

Comment:
```suggestion
In Terraform MongoDB Atlas Provider v1.10.0, we've focused on refining the management of MongoDB Atlas Programmatic API Keys (PAK) for enhanced sustainability and ease of use over the long term. Rather than embedding the PAK project assignment lifecycle in the project resource, as done previously, we've implemented a more efficient handling process that offers substantial improvements. This guide aims to provide you with a smooth transition from the initial release version (v1.8.0) to this latest release (v1.10.0).
```